### PR TITLE
Add http method to output

### PR DESCRIPTION
**Pull request by sdepold**
Great project :)

This PR prepends the http method to the requested URL.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,18 +22,20 @@ function time(start) {
 
 function defaultHandler(request, options, cb) {
 	var url = (typeof options === 'string' ? urlParse(options) : options).href;
+	var method = '[' + options.method + ']';
```
Comment (floatdrop) [inline index.js]: `options.method` can be `undefined` quite often

Comment (sdepold) [inline index.js]: Ah ok. Are you fine with the idea in general? I would add fallback to get then

Comment (Qix-) [inline index.js]: Just do `options.method || 'GET'`. That is accurate according to the API.

Comment (sdepold) [inline index.js]: fixed!

**Comment by floatdrop** [comment]
Sweet, thanks!

**Pull request merged**